evm/lightchain_shell.py

```diff
@@ -74,3 +76,81 @@ def cleanup():
 
 
 atexit.register(cleanup)
+
+# This is to start IPython or Python REPL.
+
+def ipython_shell_1(namespace=None, banner=None, debug=False):
```

Comment:
Should we drop the banner and namespace args from all those functions and add it if/when we have a use for it?

Reply:
I think we can keep it as None for now. If we ever need it we can just pass it w/o much modification to the existing code.